validator/src/commands/run/args.rs

```diff
@@ -1737,6 +1744,11 @@ mod tests {
                     max_request_body_size: Some(MAX_REQUEST_BODY_SIZE),
                     ..JsonRpcConfig::default()
                 },
+                pub_sub_config: PubSubConfig {
+                    worker_threads: 4,
+                    notification_threads: Some(NonZeroUsize::new(get_thread_count()).unwrap()),
+                    ..PubSubConfig::default()
```

Comment:
nit: Can you use `PubSubConfig::default_for_tests()` instead ?

Also, the hard-coding of the default above is not ideal / kind of brittle, but problem for another PR